src/langs/langs.js

```diff
@@ -16,4 +17,5 @@ export default {
   lv,
   nl,
   zh,
+  tw,
```

Comment:
```suggestion
  'zh-TW': tw,
```
As `tw` is not a primary language tag but a regional subtag we need to import it as `zh-TW` which then will be sent by browsers set to "Chinese (Traditional)" while "Chinese (Simplified)" will result in `zh-CN` which then will fall back to `zh` as we don't have an extra translation for `zh-CN`…

Additionally this required a change in setting the locale of the Vue18n module which I already added to the `master`: ff91301